exonum/CHANGELOG.md

```diff
@@ -18,9 +18,13 @@ All notable changes to this project will be documented in this file. The project
   around a type (e.g., `struct QuirkyI32Key(i32)`) and implement `StorageKey` for it
   using big endian encoding. Then, use the wrapper instead of the int type in indices.
   See the unit tests for `StorageKey` for an example. (#443)
+- `Transaction` `execute` method now returns  `TransactionStatus` that is stored in the blockchain and can be accessed
```

Comment:
`ExecutionResult` instead of `TransactionStatus`? Also, there is an obsolete space before `TransactionStatus`; and I would use `Transaction::execute` instead of 2 separate words.